charts/hono/values.yaml

```diff
@@ -1791,12 +1855,30 @@ dataGridExample:
   # imageName contains the name (including tag)
   # of the container image to use for the example data grid.
   imageName: "infinispan/server-native:13.0"
-  # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # [DEPRECATED: use probes instead] livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
   # configuration property of the component's liveness probe.
+  # The value of the top level "livenessProbeInitialDelaySeconds" property will be used if not set.
   livenessProbeInitialDelaySeconds: 300
-  # readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # [DEPRECATED: use probes instead] readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
   # configuration property of the component's readiness probe.
+  # The value of the top level "readinessProbeInitialDelaySeconds" property will be used if not set.
   readinessProbeInitialDelaySeconds: 30
+  # probes allows you to overwrite the global probes config values for this specific component
+  probes:
+    livenessProbe:
+      httpGet:
+        path: "/rest/v2/cache-managers/routing-info/health/status"
+        port: "hotrod"
+      failureThreshold: 5
+      successThreshold: 1
+      timeoutSeconds: 3
+    readinessProbe:
+      httpGet:
+        path: "/rest/v2/cache-managers/routing-info/health/status"
+        port: "hotrod"
+      failureThreshold: 5
+      successThreshold: 1
+      timeoutSeconds: 3
```

Comment:
IMHO we should set the initialDelaySeconds in the probes here and leave the deprecated properties empty. That way we also provide an example of how to set the values in the desired way ...